lib/config/types.ts

```diff
@@ -331,3 +331,22 @@ export type RenovateOptions =
   | RenovateBooleanOption
   | RenovateArrayOption
   | RenovateObjectOption;
+
+export interface PackageRuleInputConfig extends Record<string, any> {
```

Comment:
```suggestion
export interface PackageRuleInputConfig extends Record<string, unknown> {
```

Can we use `unknown` here?